src/app/misc/Feedback.tsx

```diff
@@ -0,0 +1,88 @@
+import { IconButton } from '@chakra-ui/button';
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverHeader,
+  PopoverBody,
+  Portal,
+  Center,
+  Text,
+  Box,
+  Link,
+  VStack,
+  Image,
+} from '@chakra-ui/react';
+import { PropsWithChildren, useState } from 'react';
+import { VscFeedback, VscChevronDown } from 'react-icons/vsc';
+
+type Props = {
+  src: string;
+  href: string;
+};
+
+function FeedbackElement({ href, src, children }: PropsWithChildren<Props>): JSX.Element | null {
+  return (
+    <Link href={href} target="_empty" _hover={{ textDecoration: 'none' }}>
+      <VStack borderRadius="xl" bg="white" mt="10px" shadow="md">
+        <Center bg="blue.300" borderTopRadius="xl" maxH="60px" objectFit="cover" overflow="hidden">
+          <Image src={src} h="100%" objectFit="cover" overflow="hidden" />
+        </Center>
+        <Box px="20px" pb="20px">
+          <Text>{children}</Text>
+        </Box>
+      </VStack>
+    </Link>
+  );
+}
+
+export function Feedback(): JSX.Element | null {
+  const [isOpen, setIsOpen] = useState(false);
+  const open = () => setIsOpen(!isOpen);
+
+  return (
+    <Popover closeOnEsc={false} isOpen={isOpen} autoFocus={false}>
+      <PopoverTrigger>
+        <IconButton
+          colorScheme="blue"
+          isRound
+          aria-label="Feedback"
+          size="lg"
+          fontSize="25px"
+          icon={isOpen ? <VscChevronDown /> : <VscFeedback />}
+          shadow="dark-lg"
+          onClick={open}
+          _focus={{
+            shadow: 'dark-lg',
+          }}
+        />
+      </PopoverTrigger>
+      <Portal>
+        <PopoverContent pos="sticky" mr="25px" maxH="500px" overflowY="scroll">
+          <PopoverHeader bg="blue.500" color="white" fontWeight="bold" textAlign="center">
+            Feedback
+          </PopoverHeader>
+          <Box px="15px" color="gray.600">
+            <PopoverBody textAlign="left">
+              Clockwork is in the early stages of development and feedback is greatly appreciated! <br />
```

Comment:
Instead this could possible say? 
`Feedback is greatly appreciated during our early stages of development! <br /> Share your thoughts here:`

I like this as:
1. Feedback is the first word in the sentence
2. It is more direct